File: nexobytes_edutech/lib/classes/timetable.dart
class Timetable {
  String id;
  String timetable;
  int createdAt;
  String createdBy;
  String classId;
  String className;
  int date;

  Timetable({
    required this.id,
    required this.timetable,
    this.createdAt=0,
    this.createdBy="",
    this.classId = "",
    this.className="",
    required this.date,
  });
}

File: server/models/timetable.js
const mongoose = require("mongoose");

const Schema = mongoose.Schema;

const timetableSchema = new Schema({
	classId: { type: String, required: true },
	className: { type: String, required: true },
	createdAt: { type: Number, required: true },
	createdBy: { type: String, required: true },
	date: { type: Number, required: true },
	timetable: { type: String, required: true },
});

module.exports = mongoose.model("Timetable", timetableSchema);
